src/apps/statements/tests/storingStatements/queriables/getMetadataFromStatement/getDurationMetadata.test.ts

```diff
@@ -0,0 +1,37 @@
+import * as assert from 'assert';
+import { assign } from 'lodash';
+
+import { getDurationMetadata } from '../../../../service/storeStatements/queriables/getMetadataFromStatement/getDurationMetadata';
+import { statementDefaults } from './fixtures/statements.fixture';
+
+describe(
+  'Retrieve duration metadata from statement',
+  () => {
+    it(
+      'should return result duration statement',
+      () => {
+        assert
+          .equal(
+            getDurationMetadata(
+              assign(
+                {},
+                statementDefaults,
+                {
+                  result: {},
+                },
+              ),
+            ),
+            false,
+          );
+
+        assert
+          .deepEqual(
+            getDurationMetadata(statementDefaults),
+            {
+              'https://learninglocker&46;net/result-duration': { seconds: 37080306 },
+            },
+          );
```

Comment:
I think code with this much indentation could be written more readably with a variable as shown below. If you agree, please change other occurrences of code like this in this pull request. If you disagree, I'd be interested to understand your reason, I do appreciate that this adds more variables to scope which may reduce performance, but I think that is negligible compared to the readability benefit, especially in test code.

```ts
const actualMetadata = getDurationMetadata(statementDefaults);
const expectedMetadata = {
  'https://learninglocker&46;net/result-duration': { seconds: 37080306 },
};
assert.deepEqual(actualMetadata, expectedMetadata);
```

Reply:
Actually I agree with you 🙂The main argument from my side is performance, you're right but readability is also one of the main rules for writing tests

Reply:
But in this case you need to think a lot of how you should name a big number of variables 😄 
```
const actualEmptyMetadataFromEmptyResult = ...
const actualEmptyMetadataFromInvalidStatement = ...
const ... (picture below)
```

![14608107_1180665285312703_1558693314_n](https://user-images.githubusercontent.com/6347345/71752112-85974480-2e86-11ea-8895-c3b52112e0a1.jpg)